lib/vanity/adapters/redis_adapter.rb

```diff
@@ -87,7 +87,7 @@ def metric_track(metric, timestamp, identity, values)
 
       def metric_values(metric, from, to)
         single = @metrics.mget(*(from.to_date..to.to_date).map { |date| "#{metric}:#{date}:value:0" }) || []
-        single.map { |v| [v] }
+        single.map { |v| [v.to_i] }
```

Comment:
Just to clarify, this would be the correct value, just in the wrong type? So it previously returned "2" and now it will return 2?